nitrite.go

```diff
@@ -62,11 +63,11 @@ type VerifyOptions struct {
 	CurrentTime time.Time
 }
 
-type coseHeader struct {
+type CoseHeader struct {
```

Comment:
I need this to be public so that I can make this structure in https://github.com/blocky/nitriding/blob/4830bbcf7c5f6e301ebdc63c1048280031a0e4f1/attester.go#L100